tests/runner/src/main.rs

```diff
@@ -30,6 +33,23 @@ enum Wal {
     Kafka,
 }
 
+// add a group to ensure that all server addresses are set together
+#[derive(clap::Args, Debug, Clone, Default)]
+#[group(multiple = true, requires_all=["server_addr", "pg_server_addr", "mysql_server_addr"])]
+struct ServerAddr {
+    /// Address of the grpc server.
+    #[clap(short, long)]
+    server_addr: Option<String>,
+
+    /// Address of the postgres server. Must be set if server_addr is set.
+    #[clap(short, long, requires = "server_addr")]
+    pg_server_addr: Option<String>,
+
+    /// Address of the mysql server. Must be set if server_addr is set.
+    #[clap(short, long, requires = "server_addr")]
+    mysql_server_addr: Option<String>,
+}
+
```

Comment:
Which would be used by default without a protocol interceptor?

Reply:
I guess the default is the grpc server. @J0HN50N133 Could you confirm this?